### ISSUE TITLE
can't be used with melonloader

### ISSUE THREAD

**Issue by catnotadog** [kruumy/EasyIMGUI#1]
can't be used with melonloader, as you're using net standard

**Comment by kruumy** [comment]
I don't know if you are missing something. All three projects are using netstandard2.0. Furthermore, I have done testing on multiple melonloader games and it works fine. Reminder that you have to put libs in the UserLibs folder in your melonloader installation. I do not know if it would work by just putting the dlls in the mods folder.

**Comment by catnotadog** [comment]
I put the dlls into userlibs, still didn't work

**Comment by catnotadog** [comment]
also i will add that the game I'm modding is il2cpp

**Comment by catnotadog** [comment]
ty

**Comment by catnotadog** [comment]
btw do you have discord?

**Comment by kruumy** [comment]
i found the issues. There was a small bug in EasyIMGUI.MelonLoader.Interface.Menu.Toggle() where the toggling would just not work. I just commited the fix. 
I did my testing on Bloons TD 6 which uses net6. In that version of the UnityEngine.IMGUIModule, GUI.WindowFunction is defined completely differently. So only the Window control wont work. I do not know if GUI.WindowFunction is different between mono and il2cpp games or that is just a difference between .net6 il2cpp games (which btd6 is a .net6 il2cpp game). I need to test more, but for now only the window control should be not working.
